virttest/utils_misc.py

```diff
@@ -1609,6 +1610,69 @@ def get_cpu_info(session=None):
     return cpu_info
 
 
+def remote_pkg_install(pkg_list, session, timeout=300):
+    """
+    Try to install packages on remote system with session.
+
+    :param pkg_list: list of packages
+    :param session: session Object
+    :param timeout: timeout for install with session
+    :return: True if all packages installed, False if any error
+    """
+    if not session or not isinstance(pkg_list, list):
+        raise exceptions.TestError("Parameters exception on session or "
```

Comment:
it could be used separately, so the check is enforced. I hope remote install could use the new remote function, while update exist yum_install to package_install in the test cases.